### ISSUE TITLE
knife-vra: "Invalid filter" submitting request

### ISSUE THREAD

**Issue by stuartpreston** [chef-partners/knife-vrealize#36]
Hi folks, I just upgraded to v2.7.0 and got this error:
`ERROR: Vra::Exception::RequestError: Unable to submit request: Invalid filter '(userName eq "mcascone" and domain eq null)': Invalid filter value beginning at position 38`

Moments prior to upgrading, i successfully ran the spin 3 times with no problems.

My command:
` knife vra server create $catitem -N 'ICERemote-4' -r 'role[ice-remote-uat]'`

_Originally posted by @mcascone in https://github.com/chef-partners/kitchen-vra/issues/33#issuecomment-492740422_

**Comment by stuartpreston** [comment]
@mcascone I spotted your comment on the _closed_ *kitchen-vra* issue  and have moved it here for tracking. Are you able to provide a bit more information about your environment (such as knife version, how you are specifying your credentials, domain and tenant in any configuration?). Thanks.

**Comment by mcascone** [comment]
Thanks Stuart, I had meant to update that yesterday. All I had to do to fix was add my @domain.com to my username variable. 

Still, it was a change from the prior version. 

I can add all the usage details when I get in front of my computer later, if you still need. 

Thanks for being so supportive!
Max

**Comment by stuartpreston** [comment]
I thought it would be something like that. Was the `@domain.com` part from an SSO setup (in which case I'm not sure how it worked before) or was it the same as the tenant name?  If the latter, it's possible I can put out a fix to get the 'old' behavior back. If the former, then it'll probably be a readme update in this repo. Thanks!

**Comment by mcascone** [comment]
Yes, we use SSO, but as far as I can tell, it was working fine the morning that i upgraded, and then broke after the upgrade. That's not to say there's no chance I wasn't doing something else dumb that this change just exposed. Or one of my environment vars was set wrong, and i hadn't noticed, etc. So i wouldn't worry about it too much, but the fact is something changed that maybe just needs to be highlighted.
I'm just seeing now the detail of your question, the `@domain` is sso, not the tenant name.
